packages/apps/braneframe/vite.config.ts

```diff
@@ -6,7 +6,7 @@ import { sentryVitePlugin } from '@sentry/vite-plugin';
 import { defineConfig } from 'vite';
 import { ConfigPlugin } from '@dxos/config/vite-plugin';
 import ReactPlugin from '@vitejs/plugin-react';
-import { ThemePlugin } from '@dxos/react-components/plugin';
+import { ThemePlugin } from '@dxos/aurora-theme/plugin';
```

Comment:
should `@dxos/aurora` re-export `@dxos/aurora-theme` and the plugin?

Reply:
I wasn't sure about the plugin… worth discussing, and would make for an easy follow-up.

Reply:
Since it's almost a requirement to use the plugin, I think it makes sense to include that. Without it you need to install both anyways.

Reply:
Sounds reasonable. Since this is so huge, I'd like to merge this first, then will make that adjustment in a follow-up.